app/services/event_driven_services.py

```
@@ -0,0 +1,766 @@
+                
+                # Check if events are flowing (if not, trigger fallback)
+                if await self._should_run_fallback(service_name):
+                    logger.debug(f"Running fallback for {service_name}")
+                    
+                    # Check resources before fallback processing  
+                    if await self._can_process_events(config["priority"]):
+                        await config["handler"](None, "fallback")
+                
+                await asyncio.sleep(interval)
+                
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)